client/src/components/PastasList.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+const PastasList = (props) => {
+  const [pastas, setPastas] = useState([]);
+
+  const getPastas = async () => {
+    try {
+      const response = await fetch(`/api/v1/pastas`);
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const body = await response.json();
+      setPastas(body.pastas);
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`);
+    }
+  };
+
+  useEffect(() => {
+    getPastas();
+  }, []);
+
+  const pastaListItems = pastas.map((pasta) => {
+    return (
+      <div className="pasta-tile" key={pasta.id}>
```

Comment:
`Link` elements can take a `className` so that we don't have to add unnecessary `div`'s